README.md

```diff
@@ -11,7 +11,8 @@ The Arcade Learning Environment (ALE) -- a platform for AI research.
 ### To compile:
 
 ```
-$ cmake -DUSE_SDL=ON -DUSE_RLGLUE=OFF -DBUILD_EXAMPLES=ON .
+$ mkdir build && cd build
```

Comment:
Not everyone uses a `build` directory, but I agree it makes for much cleaner output.